### ISSUE TITLE
Request for write access to manual

### ISSUE THREAD

**Issue by DaveSeidel** [csound/manual#132]
I have a commit that fixes some typos and has a couple other copyedits for the OSClisten and slicearray opcodes. May I be granted write access to the repo? I did not make a new branch and submit a PR, because there's nothing in the README file stating what the usual procedure is for this repo, but I would be happy to do so if that's the standard here.

**Comment by DaveSeidel** [comment]
Never mind, will do this as a pull request, which is better form.

**Comment by DaveSeidel** [comment]
reopening because I don't have permission to push a new branch

**Comment by jpffitch** [comment]
A pull request is probably best, or a patch sent to me.  Will look at adding you to the manual maintainers if you prefer

**Comment by DaveSeidel** [comment]
I can't push a new branch, so I can't make a pull request. I don't need full permissions, but if I can get permission to push a branch, then I will submit a pull request.

**Comment by DaveSeidel** [comment]
Thanks, just saw the invitation and responded.

**Comment by DaveSeidel** [comment]
@kunstmusik Maybe there's another way to do it, but on all the Git projects I have worked on (and I've been doing this on a daily basis through my job for several years now), a PR is a request to merge one branch into another. @jpffitch has granted me the access to do this, so it's now a moot point, but I'm always up for learning something I didn't know before.

**Comment by kunstmusik** [comment]
@DaveSeidel Right, but you can do a fork, then do work in the fork, including make new branches in the fork, then do the PR and say "from branch x in my fork to branch y in the upstream".  Upstream can then clone/test from the branch in fork before merging in the PR.  At least, that's what I remember.  (I will make a fork of the manual now to confirm.)

**Comment by DaveSeidel** [comment]
OK, right, forgot about that because I rarely fork, so it's never become part of my git workflow. Seems a bit heavy for small fixes like this, but yes, that's an option.

**Comment by kunstmusik** [comment]
Tested and it works, see https://github.com/csound/manual/pull/134 as an example of fork and PR.  Yes, typically fork and PR is the way to do it so that one doesn't have to add people to projects to accept contributions, plus it forces review to get changes in.  (Anyways, I don't think anyone is going to complain having you be a part of the Manual maintainers group. :) )

**Comment by DaveSeidel** [comment]
Thanks, and of course that makes sense. I guess I've been using Csound so long that it doesn't seem like a "foreign" project to me, even though I haven't been a direct contributor. Rest assured that I have plenty of git experience (though 99% of it isn't visible, since it's on private GitHub and Gitlab repos) and and won't abuse the privilege.

**Comment by kunstmusik** [comment]
No worries, and perhaps we should add some kind of mention of fork/PRs or point to the relevant documentation in our README.md.

**Comment by DaveSeidel** [comment]
And I may as well add add, I worked as a technical writer before I became a professional developer, and continue to work or and review software documentation as part of my day-to-day work.

**Comment by DaveSeidel** [comment]
Yes, that would be good.
